src/logic/tokens/store/selectors/index.ts

```diff
@@ -1,8 +1,11 @@
+import { Map } from 'immutable'
 import { createSelector } from 'reselect'
 
 import { TOKEN_REDUCER_ID } from 'src/logic/tokens/store/reducer/tokens'
+import { Token } from '../model/token'
+import { GnosisState } from 'src/store'
 
-export const tokensSelector = (state) => state[TOKEN_REDUCER_ID]
+export const tokensSelector = (state: GnosisState): Map<string, Token> => state[TOKEN_REDUCER_ID]
```

Comment:
this type should be created in a token reducer, e.g `type TokensState = Map<string, Token>`

and then imported here, so if we changed the reducer we update the type in one place